samples/application_development/sysbuild/with_mcuboot/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(sample_with_mcuboot)
+
+# Verify that this samnple is built through sysbuild to ensure MCUboot is
+# automatically included and that sample specific MCUboot configurations are
+# used when building MCUboot.
+test_sysbuild()
```

Comment:
Seems like REQUIRED should be set for this sample

Reply:
In this particular case I actually prefer to let users play / build sample without a fatal error.

That means they can actually build and flash the sample and see that it is located at start of flash when not using sysbuild, and when they build the sample with sysbuild, the image starts at an offset.

For example for the nrf52840dk, the start address without sysbuild is `0x0` (nil), and with sysbuild `0xc000`.

That way they are able to see the difference between with / without sysbuild, but of course still see the warning when not using sysbuild. Some which is not possible with `REQUIRED`.